src/scripts/modules/add_destination.js

```diff
@@ -33,10 +32,10 @@ function bindEvents() {
     //get the index of the destination pushed to the destinationsList and publish it
     const itemIndex = destinationsList.length - 1;
     events.publish("destinationAdded", itemIndex);
-    // storeNew();
+    storeNew();
     form.reset();
   });
-  // document.addEventListener("DOMContentLoaded", loadNew);
+  document.addEventListener("DOMContentLoaded", loadNew);
```

Comment:
Do you think this is the best place to add this? Based on your flow diagram you said:

> then loading it (the stored list) into the destination list when the page loads

So, this should be done in the `destinations` module, right? We should get the stored list and it needs to be rendered as it is done with the `destinationsList`.

Once again, with this tips, try to solve it yourselves again.

If you can't here's an partial solution:

<details>

```js
let storedList;

// ...

// create a new method to get the list!
function getStoredList() {
  // use double piple to fallback to empty array if the list is empty 
  const storedListLS = localStorage.getItem("destinations") || [];

  storedList  = JSON.parse(storedListLS)
}

render() {
  // loop through both destinationsList and storedList
  [...destinationsList, ...storedList].forEach( /* ... */)
}

function init() {
  cacheDom();
  bindEvents();
  getStoredList(); // call the new event to get the stored destinations!
  render();
}

```

</details>